packages/server/src/db/index.js

```diff
@@ -822,14 +832,28 @@ async function enhanceGrantData(tenantId, data) {
     );
     const interestedBy = await getInterestedAgencies({ grantIds: data.map((grant) => grant.grant_id), tenantId });
 
+    const followedByQuery = knex(TABLES.agencies)
+        .join(TABLES.users, `${TABLES.agencies}.id`, '=', `${TABLES.users}.agency_id`)
+        .join(TABLES.grant_followers, `${TABLES.users}.id`, '=', `${TABLES.grant_followers}.user_id`)
+        .whereIn('grant_id', data.map((grant) => grant.grant_id))
+        .andWhere('agencies.tenant_id', tenantId);
+    const followedBy = await followedByQuery.distinct(
+        `${TABLES.grant_followers}.grant_id`,
+        `${TABLES.grant_followers}.user_id`,
+        `${TABLES.agencies}.name as agency_name`,
+        `${TABLES.agencies}.abbreviation as agency_abbreviation`,
+    );
```

Comment:
should I put this query behind the feature flag as well? It won't be displayed to the user if the feature flag is off, but this is some extra work we are doing here if feature flag is off